windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,62 @@
+[id="windows-containers-release-notes-3-x"]
```

Comment:
This needs a known issue section, where we should mention:
> Users running WMCO 2.0.2 and below will have to uninstall the operator and reinstall WMCO 3.0.0 to work around https://bugzilla.redhat.com/show_bug.cgi?id=1983153

cc: @mansikulkarni96